### DISCUSSION TITLE
Empty required field should generate a message on save

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16775] [Feature Requests]
In an M2M collection, I have a required field.
When the user leaves it empty and tries to save, the App is silent just blocking the saving

- It would be nice to tell him a field is empty and which one.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
